core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/documents/CustomizeViewPlugin.java

```diff
@@ -0,0 +1,281 @@
+
+/*
+ * Copyright (C) 2003-2018 eXo Platform SAS.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.exoplatform.ecm.webui.component.explorer.documents;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.jcr.Node;
+import javax.jcr.Property;
+import javax.jcr.RepositoryException;
+import javax.jcr.Session;
+
+import org.exoplatform.commons.api.settings.SettingService;
+import org.exoplatform.commons.api.settings.SettingValue;
+import org.exoplatform.commons.api.settings.data.Context;
+import org.exoplatform.commons.api.settings.data.Scope;
+import org.exoplatform.container.configuration.ConfigurationManager;
+import org.exoplatform.container.xml.InitParams;
+import org.exoplatform.container.xml.ObjectParameter;
+import org.exoplatform.services.cms.BasePath;
+import org.exoplatform.services.cms.impl.DMSConfiguration;
+import org.exoplatform.services.cms.impl.DMSRepositoryConfiguration;
+import org.exoplatform.services.cms.templates.TemplateService;
+import org.exoplatform.services.cms.views.ViewConfig;
+import org.exoplatform.services.cms.views.ViewConfig.Tab;
+import org.exoplatform.services.cms.views.impl.ManageViewPlugin;
+import org.exoplatform.services.jcr.RepositoryService;
+import org.exoplatform.services.jcr.core.ManageableRepository;
+import org.exoplatform.services.jcr.ext.hierarchy.NodeHierarchyCreator;
+
+/**
+ * Customize existing view of ECMS Document Explorer.
+ * It allows to add buttons in existing tabs of Document Explorer views.
+ * 
+ * Created by The eXo Platform SAS.
+ * 
+ * @author <a href="mailto:pnedonosko@exoplatform.com">Peter Nedonosko</a>
+ * @version $Id: CustomizeViewPlugin.java 00000 Mar 2, 2016 pnedonosko $
+ * 
+ */
+public class CustomizeViewPlugin extends ManageViewPlugin {
+
+  /** The Constant PROVIDER_SCOPE_NAME. */
+  protected static final String        PROVIDER_SCOPE_NAME   = "customize.view.provider".intern();
+
+  /** The Constant VIEW_CUSTOMIZED_PARAM. */
+  protected static final String        VIEW_CUSTOMIZED_PARAM = "view.customized.param".intern();
```

Comment:
```suggestion
  protected static final String        VIEW_CUSTOMIZED_PARAM = "documents.views.customized".intern();
```
No need to add the "param" suffix IMO since we know it is a param.

Reply:
Changed: https://github.com/exodev/ecms/pull/510/commits/fc6a1cfda3a35149448c76a2a7f95f4f96dbd598